docs/spec/index.html

```diff
@@ -1,740 +1,952 @@
+      asynchronous in nature. This means that certain rules should be
+      followed to ensure that a reciever of data using this
+      specification can assemble a graph of connected data elements and
+      have a high degree of certainty (unless they are missing data)
+      that the data they are looking at is the most current information,
+      assembled in the right order, and is the best known state at a
+      given time. The following section details rules for handling
+      identifiers and references to data to prevent issues with
+      informational processing.
+    </p>
+    <section class="informative">
+      <h2>Identifiers</h2>
       <p>
-        The <a href="#api-spec">Open API Specification</a> section covers each
-        operation and provides example inputs and expected responses.
+        Identifiers in objects exchanged using this specification are
+        particularly important, not least in that they uniquely identify
+        an object or network transaction, but also, given the
+        asynchronous nature of this mode of data exchange, that they
+        present an area where various attacks could arise by sending
+        data with identifiers already in use with malicious intent to
+        confuse a receiver of the data.
+      </p>
+      <p>
+        Identifiers conformant with this spec <em>MUST</em> be
+        [[rfc3986]] conformant URIs. Unless otherwise noted, an
+        identifier <em>MAY</em> be a DID identifier per [[did-core]], a
+        UUID v4 per [[rfc4122]], or a [[URL]] that identifies a resource
+        directly.
       </p>
-
       <section class="informative">
-        <h3>CCG Standards Stack</h3>
-        <img src="resources/standards-stack.png" alt="standards-stack" width="100%">
+        <h2>Presentations</h2>
         <p>
-          The above diagram illustrates how traceability-interop fits into the
-          larger family of W3C CCG standards.
+          Presentations in this spec are
+          <a
+            href="https://w3c-ccg.github.io/traceability-vocab/#TraceablePresentation">Traceable
+            Presentations</a>, which contain several notable identifiers
+          that can be used for correlation, retrieval, and business rule
+          processing of data.
         </p>
+        These identifiers are:
         <ul>
           <li>
-            At the <em>Business</em> layer, supply chain actors are exchanging
-            data in support of exchanged goods and services.
-          </li>
-          <li>
-            The <em>Trust</em> layer is supported by actors signing and
-            verifying data as Verifiable Credentials.
-          </li>
-          <li>
-            The <em>Process</em> defined standardized APIs for how to exchange
-            Verifiable Credentials (VC-API) and Verifiable Presentations
-            (trace-interop).
+            `TraceablePresentation.id` — The unique ID of the
```

Comment:
I'll note that this syntax has confused folks before, but I understand the intention.